model/testing/src/icon4py/model/testing/helpers.py

```diff
@@ -69,6 +78,27 @@ def allocate_data(backend, input_data):
     return input_data
 
 
+def match_marker(
```

Comment:
the comment to give this function a better name got detached, so I am repeating it.